learning/tour-of-beam/terraform/api_enable/main.tf

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+resource "google_project_service" "required_services" {
+  project   = var.project_id
+  for_each  = toset([
+    "cloudresourcemanager",
+    "iam",
+    "cloudbuild",
+    "cloudfunctions",
+    "firebase"
+  ])
+  service   = "${each.key}.googleapis.com"
+  disable_on_destroy = false
+}
+
+resource "null_resource" "enable_cf_admin" {
+  provisioner "local-exec" {
+    command = "gcloud projects add-iam-policy-binding ${var.project_id} --member=user:$(gcloud config get-value core/account) --role=roles/cloudfunctions.admin --condition=None"
```

Comment:
Is there other way to do this?

Reply:
Null_resource has been removed, and replaced with google_project_iam_binding